src/app/_components/BorderHighlight/index.module.scss

```diff
@@ -0,0 +1,49 @@
+@use '@scss/common.scss' as *;
+
+.container {
+  position: relative;
+}
+
+.backgroundContainer {
+  position: relative;
+  border: 1px solid var(--theme-border-color);
```

Comment:
I was thinking this would be set as a prop since we turned it into a component, i.e. highlightColor: 'default' | 'success'

Then you could potentially re-use this component inside the ExtendedBackground component and remove the related css/html from there.